include/uk/weak_refcount.h

```diff
@@ -0,0 +1,87 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+/* Strong/Weak reference counting */
+
+#ifndef __UK_WEAK_REFCOUNT_H__
```

Comment:
We usually have the following header guards layout (no defintion to `1` and closing `#endif` repeats the guard name with a comment):
```c
#ifndef __UK_WEAK_REFCOUNT_H__
#define __UK_WEAK_REFCOUNT_H__
```
...
```c
#endif /* __UK_WEAK_REFCOUNT_H__ */
```